thavalon-server/src/game/messages.rs

```diff
@@ -128,6 +128,13 @@ pub enum Message {
         winning_team: Team,
         roles: HashMap<String, RoleDetails>,
     },
+
+    /// Information sent to lovers specific to their role after each mission.
+    LoversInfo {
+        lone_lover: bool,
+        lover_on_mission: bool,
+        lover_identity: String,
```

Comment:
Instead of a struct, could we use a new enum here? You could then have the LoverOnMission take a tuple struct with a string that could contain lover identity.

Reply:
Talking to Jared, could we actually make the outer message type something like `ToastNotification` that takes in a notification type of LoversInfo? Might need to discuss that more fully before making the change.

Reply:
left this alone for now, we should discuss

Reply:
changed LoversInfo to Toast message with new string messages